lib/ddtrace/contrib/rails/patcher.rb

```diff
@@ -23,7 +22,6 @@ module Patcher
         option :trace_agent_port, default: Datadog::Writer::PORT
```

Comment:
we should remember to remove these since we have to configure the Tracer using the `Tracer#configure` with the newer API.